CMakeLists.txt

```diff
@@ -473,3 +473,43 @@ ExternalProject_Add(aws-sdk-cpp
     -DCMAKE_INSTALL_PREFIX:PATH=${TRITON_THIRD_PARTY_INSTALL_PREFIX}/aws-sdk-cpp
   PATCH_COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/tools/install_src.py --src <SOURCE_DIR> ${INSTALL_SRC_DEST_ARG}
 )
+
+#
+# Build OpenTelemetry C++ library
+# [FIXME] Remove WIN flag when windos build works
```

Comment:
*windows

Reply:
Also, please add Jira to the FIXME so we know this is tracked.